src/skills-builder/skills-builder-modal/select-preferences/JobTitleAutosuggest.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form,
+} from '@edx/paragon';
+import { logError } from '@edx/frontend-platform/logging';
+import { useAlgoliaSearch } from '../../utils/search';
+
+const JobTitleAutosuggest = ({ onSelected }) => {
+  // We only need to destructure the job search index here
+  const [,, jobSearchIndex] = useAlgoliaSearch();
+
+  const [jobInput, setJobInput] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [jobList, setJobList] = useState([]);
+
+  const queryForJobs = (query = '') => {
+    jobSearchIndex.search(query).then(({ hits }) => {
+      setJobList(hits);
+      setIsLoading(false);
```

Comment:
Is the utility function from `search.jsx` not good enough here? This seems like some redundant code. Maybe I don't understand the use case here.

Reply:
From what I could tell, the utility functions don't return jobs based on a search query. There is one that returns job information but it requires job names. The query in this PR returns a list of job names to choose from based on whatever string you provide it. Perhaps I missed something?